titus-ext/cassandra/src/main/java/com/netflix/titus/ext/cassandra/store/CassandraLoadBalancerStore.java

```diff
@@ -122,9 +161,27 @@ public CassandraLoadBalancerStore(CassandraStoreConfiguration configuration,
         this.loadBalancerStateMap = new ConcurrentHashMap<>();
         this.jobToAssociatedLoadBalancersMap = new ConcurrentHashMap<>();
 
-        this.getAllJobIdsStmt = session.prepare(GET_ALL_JOB_IDS);
-        this.insertLoadBalancerStmt = session.prepare(INSERT_JOB_LOAD_BALANCER);
-        this.deleteLoadBalancerStmt = session.prepare(DELETE_JOB_LOAD_BALANCER);
+        this.selectAssociations = session.prepare(GET_ALL_ASSOCIATIONS).setConsistencyLevel(ConsistencyLevel.LOCAL_QUORUM);
+        this.insertAssociation = session.prepare(INSERT_ASSOCIATION).setConsistencyLevel(ConsistencyLevel.LOCAL_QUORUM);
+        this.deleteAssociation = session.prepare(DELETE_ASSOCIATION).setConsistencyLevel(ConsistencyLevel.LOCAL_QUORUM);
+
+        this.selectTargetsForLoadBalancer = session.prepare(
+                select(COLUMN_LOAD_BALANCER, COLUMN_IP_ADDRESS, COLUMN_TASK_ID, COLUMN_STATE)
```

Comment:
Two different styles of building queries?

Reply:
I liked the type/grammar safety of the dsl builder, but didn't want to go and change everything else in this PR. Any strong preferences?